src/Testing/Http/HttpRouterTester.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tempest\Testing\Http;
+
+use Exception;
+use Tempest\Container\Container;
+use Tempest\Http\GenericRequest;
+use Tempest\Http\Method;
+use Tempest\Http\Request;
+use Tempest\Http\Router;
+use function Tempest\map;
+
+final class HttpRouterTester
```

Comment:
I'd call it `HttpTester` I think… Not sure though

Reply:
How about `HttpRequestTester`?